src/cli/index.ts

```diff
@@ -267,6 +267,7 @@ function inferCLIOptions(opts: Partial<CLIOptions>, targetLanguage: TargetLangua
     /* tslint:disable:strict-boolean-expressions */
     const options: CLIOptions = {
         src: opts.src || [],
+        srcUrls: opts.srcUrls,
```

Comment:
Tiny change. It was being parsed correctly, but not assigned to the new `options` object.